brigade.js

```diff
@@ -195,19 +197,19 @@ function release(p, tag) {
 
   console.log(cx.tasks);
   console.log(`releases at https://github.com/${p.repo.name}/releases/tag/${tag}`);
-  return cx.run();
+  return cx;
 }
 
 // Separate docker build stage as there may be multiple consumers/publishers,
 // For example, publishing to Dockerhub and ACR below
-function goDockerBuild(project, tag) {
+function goDockerBuild(e, p) {
   // We build in a separate pod b/c AKS's Docker is too old to do multi-stage builds.
   const builder = new Job(`${projectName}-docker-build`, goImg);
 
   builder.storage.enabled = true;
   builder.env = goEnv;
   builder.tasks = [
-    `cd /src && git checkout ${tag}`,
+    `cd /src`,
```

Comment:
Is this because the sidecar already checked out the correct revision?

Reply:
@radu-matei indeed 👍